sdks/java/core/src/main/java/org/apache/beam/sdk/harness/JvmInitializer.java

```diff
@@ -33,6 +34,7 @@
  * automate this.
  */
 @Experimental
+@Internal
```

Comment:
This isn't internal, we expect people to use this and for **workers** to honor it.

Reply:
For this PR I assumed that most things were `@Internal` if the audience was runner/harness authors instead of pipeline authors (final users). Is this class expected to be used by the latter? otherwise it still fits the `@Internal` definition like `DoFnInvokers` and the others, no?

Reply:
We expect pipeline authors to write JVmInitializer and for the worker to use ServiceLoader to find them and use them.